### PR TITLE
unplug pluggable frontends

### PR DESCRIPTION
**Pull request by matallo**
Related to: #13444

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 CartoDB::Application.configure do
   # Settings specified here will take precedence over those in config/application.rb
 
-  ActiveSupport::Dependencies.autoload_paths << File::join(Rails.root, 'lib')
+  # ActiveSupport::Dependencies.autoload_paths << File::join(Rails.root, 'lib')
```
Comment (javitonino) [inline config/environments/development.rb]: Please revert

File: config/environments/development.rb [hunk 2]
```diff
@@ -53,7 +53,7 @@
   # `bundle exec thin start --threaded -p 3000 --threadpool-size 5`.
   # Check your `config/database.yml` has `pool: 50` or higher for `development`.
   # The condition excludes this from resque, since it won't work with it and it doesn't need it.
-  config.threadsafe! unless $rails_rake_task
+  # config.threadsafe! unless $rails_rake_task
```
Comment (javitonino) [inline config/environments/development.rb]: This, as well

**Comment by javitonino** [comment]
We may want to revert https://github.com/CartoDB/cartodb/pull/11183 as well.

File: tree.config.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
   "testsFolder": [
-    "lib/assets/core/test/spec/cartodb3/",
-    "lib/assets/core/test/spec/deep-insights"
+    "lib/assets/test/spec/cartodb3/",
+    "lib/assets/test/spec/deep-insights"
   ],
   "filesRegex": "spec\\.js$",
   "replacements": [
-    ["lib/assets/core", "lib/assets"],
+    ["lib/assets", "lib/assets"],
```
Comment (rubenmoya) [inline tree.config.json]: Does this line make sense anymore?

File: Gruntfile.js [hunk 4]
```diff
@@ -320,35 +302,21 @@ module.exports = function (grunt) {
...
     'cdb',
-    'copy:js_cartodb',
     'setConfig:env.browserify_watch:true',
     'npm-carto-node',
     'run_browserify',
     'concat:js',
     'jst'
   ]);
 
-  grunt.registerTask('js_builder', [
```
Comment (rubenmoya) [inline Gruntfile.js]: 🔥🔥🔥🔥🔥🔥

File: tree.config.json (hunk 3 above)
Comment (matallo) [reply tree.config.json]: removed the `replacements` alltogether

File: Gruntfile.js [hunk 5]
```diff
@@ -458,16 +423,6 @@ module.exports = function (grunt) {
     grunt.config.set(name, val);
   });
 
-  /**
```
Comment (ivanmalagon) [inline Gruntfile.js]: Are editor specs wiped out?

Comment (matallo) [reply Gruntfile.js]: I don't know why they were duplicated, they appear again some lines below affected_editor_specs

Comment (matallo) [reply Gruntfile.js]: @jesusbotella taking care of ^^^

Comment (jesusbotella) [reply Gruntfile.js]: I renamed `affected_spec` task to `test:browser`, which I think it is clearer than before.

**Comment by rubenmoya** [comment]
BOOM!! 💥

**Pull request merged**